docker-compose.yml

```diff
@@ -1,12 +1,22 @@
 version: '3'
 services:
+
     web:
-        build: ./
+        build: ./.docker/web
         container_name: svgtranslate_web
         ports:
           - ${TOOLFORGE_DOCKER_PORT}:80
```

Comment:
I don't think this is necessary, if someone needs to change the port they can extend the compose file with a `docker-compose.override.yml` file... or am I missing something?

Reply:
Is that the normal way to do things? Do configs get merged then? Could we do the same for `uid` and `gid` for volumes for instance?

Reply:
and yes, you could do it for anything and just let someone override it if they want/need to. (or create a `.env.dist` file and make them copy rename the file to `.env` and add that to gitignore)